Queue/Backend/Redis.php

```diff
@@ -233,7 +233,7 @@ private function connectIfNeeded()
     protected function connect()
     {
         $this->redis = new \Redis();
-        $success = $this->redis->connect($this->host, $this->port, $this->timeout, null, 100);
+        $success = $this->redis->pconnect($this->host, $this->port, $this->timeout, null, 100);
```

Comment:
ah, yes, it's unrelated to this pull request, will remove it.

You are right, `pconnect` is to open persistent connection using phpredis (phpredis is used in QueuedTracking if sentinel is not enabled). `pconnect` is quite useful for high traffic websites. We were using it before unix socket support, because `connect` leaves a lot of tcp connections in `TIME-WAIT` state (every request makes a new connection and does not close it at the end of request).